src/main/kotlin/no/nav/klage/search/api/controller/OppgaverListController.kt

```diff
@@ -154,6 +154,44 @@ class OppgaverListController(
         )
     }
 
+    @ApiOperation(
+        value = "Hent oppgaver satt på vent for en ansatt",
+        notes = "Henter alle oppgaver satt på vent som saksbehandler har tilgang til."
+    )
+    @GetMapping("/ansatte/{navIdent}/oppgaver/paavent", produces = ["application/json"])
```

Comment:
`/oppgaver` er en mengde, som for meg da ikke kan ha properties som f.eks. `paavent`. Hvis man vil filtrere oppgaver så tenker jeg at det er ved hjelp av query params: `/oppgaver?paavent=true` e.l. Alternativt opprette en egen ressurs/mengde som heter `/oppgaverpaavent` f.eks. Tanker? Går litt på REST-prinsipper (som vi ikke følger slavisk da). Kan være vi allerede er langt vekk fra diverse prinsipper og at det ikke har så mye å si, siden vi bare har én klient der vi blir enige om hva ting kalles :)

Reply:
Måten han gjør det på her følger bare mønsteret vi har lagt opp til nå.

Reply:
Den er grei.